src/main/java/org/dependencytrack/resources/v1/ComponentResource.java

```diff
@@ -73,7 +73,29 @@
 @Path("/v1/component")
 @Api(value = "component", authorizations = @Authorization(value = "X-Api-Key"))
 public class ComponentResource extends AlpineResource {
-
+    
+    @GET
+    @Path("/all")
```

Comment:
Should this be just "/" (or removed)? Other resources return the list of all entities from "/".

Reply:
Yes, I have removed it.